File: Examples/Java/SDK/src/main/java/com/aspose/imaging/cloud/examples/images/UpdateGifPropertiesWithoutStorage.java
package com.aspose.imaging.cloud.examples.images;

import com.aspose.imaging.cloud.examples.Utils;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class UpdateGifPropertiesWithoutStorage {

    public static void main(String... args) throws IOException {
        String input = "Sample1.gif";
        String output = "Sample2.gif";
        Path inputFile = Utils.getPath(UpdateGifPropertiesWithoutStorage.class, input);
        Path outputFile = Utils.getPath(UpdateGifPropertiesWithoutStorage.class, output);

        Integer backgroundColorIndex = 255;
        Integer colorResolution = 7;
        Boolean hasTrailer = true;
        Boolean interlaced = true;
        Boolean isPaletteSorted = true;
        Integer pixelAspectRatio = 10;
        Boolean fromScratch = false;
        String outPath = "";

        com.aspose.imaging.model.ResponseMessage sr = Utils.getImagingSdk().PostImageGif(
                backgroundColorIndex,
                colorResolution,
                hasTrailer,
                interlaced,
                isPaletteSorted,
                pixelAspectRatio,
                fromScratch,
                outPath,
                inputFile.toFile()
        );
        
        Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);

    }
}

File: Examples/Node.js/REST/images/UpdateGifPropertiesWithoutStorage.js
var utils = require("../utils.js");
var fs = require('fs');

var input_file = 'sample1.gif';
var inputPath = getPath(__filename, input_file);
var output_file = 'output.gif';
var outputPath = getPath(__filename, output_file);

var input_buffer = fs.readFileSync(inputPath);

var method = 'POST';
var request_url = 'imaging/gif?backgroundColorIndex=255&pixelAspectRatio=10';

ProcessCommandStream(
    method,
    Sign(request_url),
    input_buffer,
    function(output_buffer) {
        fs.writeFileSync(outputPath, output_buffer);
    }
);